src/SignalR/clients/ts/signalr/src/HttpConnection.ts

```diff
@@ -61,7 +61,7 @@ export class HttpConnection implements IConnection {
     private transport?: ITransport;
     private startInternalPromise?: Promise<void>;
     private stopPromise?: Promise<void>;
-    private stopPromiseResolver!: (value?: PromiseLike<void>) => void;
+    private stopPromiseResolver: (value?: PromiseLike<void>) => void = () => {};
```

Comment:
@BrennanConroy do we need to bring these changes over from 3.1 -> main?

Reply:
Should already be in main